src/devices_models/device_constructors/hydrogeneration_constructor.jl

```diff
@@ -105,6 +105,30 @@ function construct_device!(
     )
 
     add_parameters!(container, ActivePowerTimeSeriesParameter, devices, model)
+
+    if has_service_model(model)
+        time_steps = get_time_steps(container)
+        names = PSY.get_name.(devices)
+        for meta in [LOWER_BOUND, UPPER_BOUND]
+            add_expression_container!(
```

Comment:
Why are you adding the container here? We have to use `add_to_expression!` methods to handle this. The addition of the container is done inside of the `add_to_expression!` call.